fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayTabLayouts.kt

```diff
@@ -59,10 +59,20 @@ fun TabLayout(
     onTabLongClick: (TabSessionState) -> Unit,
     header: (@Composable () -> Unit)? = null,
 ) {
+    var selectedTabIndex = 0
+    selectedTabId?.let {
+        tabs.forEachIndexed { index, tab ->
+            if (tab.id == selectedTabId) {
+                selectedTabIndex = index
+            }
+        }
+    }
```

Comment:
nit: We could avoid a `var` here with something like:
```
val selectedTabIndex = tabs.indexOf(tabs.find { tab ->
    tab.id == selectedTabId
})
```

Feels a little hacky with all the nullability implied in there, but might be preferable.

Reply:
I wanted to avoid this, since doing `indexOf` + `tabs.find` causes two linear iterations over the list of tabs, but also `find` can return `-1` when the tab isn't in the list (thinking of the situation it's searching for a Private tab in the normal list) which would cause an exception.